packages/grid/x-data-grid/src/models/api/gridColumnSpanning.ts

```diff
@@ -0,0 +1,31 @@
+import { GridColumnIndex, GridCellColSpanInfo } from '../gridColumnSpanning';
+import { GridRowId } from '../gridRows';
+/**
+ * The Column Spanning API interface that is available in the grid `apiRef`.
+ */
+export interface GridColumnSpanning {
```

Comment:
We usually suffix with "Api". The same for the filename.

```suggestion
export interface GridColumnSpanningApi {
```